taxcalc/dropq/dropq_utils.py

```diff
@@ -91,6 +91,7 @@ def dropq_calculate(year_n, start_year,
         recs1p = Records(data=taxrec_df.copy(deep=True),
                          gfactors=growfactors_pre)
         # add one dollar to total wages and salaries of each filing unit
+        # to determine which records are taxpayers
```

Comment:
Hmmm `taxpayer` might be not be the most accurate term, since some people might be have credits completly offset liabilities, but technically they're still taxpayers. Quoting Dan's original documentation - "for every taxpayer we calculate the effect of adding a dollar to income, then keep track of the number of taxpayers in each bracket for whickh those additions make a difference in the tax liability. This is the count of effective cell size." So maybe rephrase a bit?